pkg/cache/context.go

```diff
@@ -681,17 +681,13 @@ func (ctx *Context) IsPodFitNodeViaPreemption(name, node string, allocations []s
 			// look up each victim in the scheduler cache
 			victims := make([]*v1.Pod, 0)
```

Comment:
Could we set the size?

Reply:
Yes, we can initialized the size first to further enhance the performance.

Reply:
Done.